testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -575,6 +575,20 @@ public void testUnionDistinctViews()
         }
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testHivePartitionViews()
+    {
+        onHive().executeQuery("DROP VIEW IF EXISTS hive_partition_view");
```

Comment:
> hive_partition_view
> test_hive_partition

Let's rename to
* test_view_partitioned_column
* test_table_partitioned_column